mod/projects/set/ltype_rtype/metric/project.rb

```diff
@@ -44,6 +44,6 @@ def project_name
 
   def metric_link values=:all
     path_args = card.project_card.filter_path_args values
-    link_to_card card.metric_card, yield, path: path_args
+    link_to_card card.metric_card, yield, path: path_args, class: "metric-color"
```

Comment:
seems like an iffy class name. maybe `metric-filtered-link`

Reply:
`metric-color` is a generic class name, I prefer to re-use CSS classes instead of creating a new one.

Reply:
ok